src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -304,5 +314,893 @@ public void Memory_ClearTest()
+            page.MemoryPanel.ResizeWindowToDiplayMemoryLabel();
+            Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("MemoryPaneEmpty"));
+        }
+
+        [TestMethod]
+        [Priority(2)]
+        public void BVT_HistoryPanel()
+        {
+
+            /// <summary>
+            /// Test Case 23480026: BVT: Standard/Scientific: Verify history panel
+            /// This automated test verifies using using the history panel in Standard mode
+            /// Note: An empty history on initial launch cannot be verified.  The automation cannot see the Windows element "HistoryEmpty."
+            /// Even though this element is there; I see it, and Inspect sees it, WinAppDriver cannot find it.
+            /// The only time the automation is successful at finding this element is after creating and deleting history items.
+            /// Will this part of the test later, once issue is solved.
+            /// </summary>
+
+            //Verify selecting history items form the history panel, and clearing the history panel
+            page.HistoryPanel.OpenHistoryPanel();
+
+            page.StandardOperators.NumberPad.Input(-3);
+            page.StandardOperators.PlusButton.Click();
+            page.StandardOperators.NumberPad.Input(-2.6);
+            page.StandardOperators.EqualButton.Click();
+
+            page.StandardOperators.NumberPad.Input(2);
+            page.StandardOperators.MinusButton.Click();
+            page.StandardOperators.NumberPad.Input(3);
+            page.StandardOperators.EqualButton.Click();
+
+            var historyItems = page.HistoryPanel.GetAllHistoryListViewItems();
+            Assert.IsTrue(historyItems[0].Text.Equals("2 Minus ( 3= Minus (1", StringComparison.InvariantCultureIgnoreCase));
+            Assert.IsTrue(historyItems[1].Text.Equals("-3 + -2.6= Minus (5.6", StringComparison.InvariantCultureIgnoreCase));
+
+            Assert.AreEqual("-1", page.GetCalculatorResultText());
+
+            Actions clickHistoryItemsw1 = new Actions(WinAppDriver.Instance.CalculatorSession);
+            clickHistoryItemsw1.Click(historyItems[1]);
+            clickHistoryItemsw1.Perform();
+
+            Assert.AreEqual("-5.6", page.GetCalculatorResultText());
+            Assert.AreEqual("-3 + -2.6=", page.GetCalculatorExpressionText());
+
+            Actions clickHistoryItemsw0 = new Actions(WinAppDriver.Instance.CalculatorSession);
+            clickHistoryItemsw0.Click(historyItems[0]);
+            clickHistoryItemsw0.Perform();
+
+            Assert.AreEqual("-1", page.GetCalculatorResultText());
+
+            page.HistoryPanel.ClearHistory();
+            Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("HistoryEmpty"));
+        }
+
+        [TestMethod]
+        [Priority(2)]
+        public void BVT_HistoryFlyout()
+        {
```

Comment:
How does the history flyout behave different than the history panel? Is it the same control that is accessed in different ways?

Reply:
The UI is interacted with in different ways